server/routes/viewingLicences/handlers/printLicence.ts

```diff
@@ -35,22 +46,30 @@ export default class PrintLicenceRoutes {
       additionalPssConditions,
       qrCode,
       htmlPrint,
+      exclusionZoneDescription,
+      exclusionZoneMapData,
     })
     logger.info(`HTML preview licence ID [${licence.id}] type [${licence.typeCode}] by user [${username}]`)
   }
 
   renderPdf = async (req: Request, res: Response): Promise<void> => {
     const { username } = res.locals.user
-    const { licence } = res.locals
+    const { licence, user } = res.locals
     const { qrCodesEnabled } = res.locals
     const { licencesUrl, pdfOptions, watermark } = config.apis.gotenberg
-
     const qrCode = qrCodesEnabled ? await this.qrCodeService.getQrCode(licence) : null
     const additionalLicenceConditions = expandAdditionalConditions(licence.additionalLicenceConditions)
     const additionalPssConditions = expandAdditionalConditions(licence.additionalPssConditions)
-    const imageData = await this.prisonerService.getPrisonerImageData(username, licence.nomsId)
+    const imageData = await this.prisonerService.getPrisonerImageData(licence.nomsId, user)
     const filename = licence.nomsId ? `${licence.nomsId}.pdf` : `${licence.lastName}.pdf`
```

Comment:
This was a bug left over from previous PRs - just correcting it here so it finds the prisoner image.